delivery_transsmart/models/transsmart_package_type.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+# © 2017 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import fields, models
+
+
+class TranssmartPackageType(models.Model):
+    _name = 'transsmart.package.type'
+
+    name = fields.Char()
+    _type = fields.Char()
```

Comment:
Do not start a field name with _. It suggest it is something private, while I think you just want to avoid a warning on using a reserved word as variable/field name. Call it something like package_type.